cmd/containerd-shim/main_unix.go

```diff
@@ -58,6 +60,13 @@ func init() {
 }
 
 func main() {
+	debug.SetGCPercent(10)
```

Comment:
looks good, but could/should these be configurable to let folks tune?

Reply:
we can add a flag on the shim post merge, i'm cool with that

Reply:
How does `-Xio.containerd.ShimMemoryUsage=keepitlow` sound?

Reply:
I'd prefer a runtime flag instead of a build time one

Reply:
@mlaventure Sorry, this was a JVM joke.

We can add a flag in a follow up, but I don't think it is important to tune for this use case.

Reply:
Ah :sweat_smile:, I don't Java ;-)

Reply:
Yes, not tuning for this use case was the point of my let it be configurable comment :-)